autoprotocol/container.py

```diff
@@ -54,10 +57,15 @@ def validate_properties(properties):
                 raise TypeError(
                     "Aliquot property {} : {} has a key of type {}, "
                     "it should be a 'str'.".format(key, value, type(key)))
-            if not isinstance(value, basestring):
+            try:
```

Comment:
Thinking about it... I think this is something we should highlight out in the changelog as a separate line-item as well